apps/bfd-data-npi/README.md

```diff
@@ -0,0 +1,25 @@
+BFD NPI to Organization Name Lookup
+=======================================
+
+This maven project produces a jar file that provides functionality to retrieve organization names given an NPI as input. This lookup is backed by a data file stored in the jar files as a resource.
+
+The data file is constructed from 
+NPI|Entity Type Code|Replacement NPI|Employer Identification Number (EIN)|Provider Organization Name (Legal Business Name)Provider Last Name (Legal Name)|Provider First Name|
```

Comment:
I think there is a pipe character missing between the org nam and the provider last name

Reply:
done